src/server.py

```diff
@@ -437,25 +405,13 @@ def run_seta_query():
 @app.route("/data/setadetails/")
 @json_response
 def run_seta_details_query():
-    # TODO: remove inactive when buildbot api queries s/inactive/priority/
-    inactive = sanitize_bool(request.args.get("inactive", 0))
```

Comment:
wait, I see the issue opened up- lets merge this code and track it with the issues.